README.md

```diff
@@ -21,23 +21,25 @@ example demonstrates the usage of HTTPLimiter for rate-limiting access
 to an http.Handler to 20 requests per path per minute with bursts of
 up to 5 additional requests:
 
-	store, err := memstore.New(65536)
-	if err != nil {
-		log.Fatal(err)
-	}
+```go
+store, err := memstore.New(65536)
+if err != nil {
+	log.Fatal(err)
```

Comment:
@beatrichartz The SVGs look awesome! Can we leave all these blocks tabbed in though? Way better for copy + paste purposes as this is the convention in Go.